src/components/sections/Header.js

```diff
@@ -23,7 +24,16 @@ const Header = props => (
         </div>
       </div>
     </Container>
-  </header>
+  </HeaderWrapper>
 );
 
+const HeaderWrapper = styled.div`
```

Comment:
Hey, @stvmachine! Thanks for the PR. 
This can be changed to styled.header`` everything else LGTM!